src/options/containers/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import App from "../components/App";
+import { promisifiedChromeStorageSyncGet } from "../../shared/chrome";
+
+const defaultOptions = {
+  json: {
+    config: "",
+    enable: false
+  },
+  prettier: {
```

Comment:
Sooo after suggesting we scope the options, I realized that options is super generic and it gets really confusing to have a nested key called options. Just want to make sure this is cool with everyone else - best to fix it now before v1!

Reply:
I feel like `prettier` explains less about what they are, especially since there are various API settings for Prettier other than `options`. `prettierOptions` could be more explicit, but personally I think the `prettier` prefix is redundant.

Reply:
`prettierOptions` sounds good to me!